dts/bindings/sensor/meas,ms56xx-common.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2024, Aaron Chan
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: |
+    TE Connectivity MS5607 and MS5611 digital pressure and temperature sensors.
+    The Datasheet is at https://www.te.com/usa-en/product-CAT-BLPS0035.html
+
+include: sensor-device.yaml
+
+properties:
+  chip:
+    type: int
+    required: true
+    description: |
+      Specifies which chip is used to determine calculation coefficients
```

Comment:
I think it would be appropriate here to have two different compatibles rather than a property

Reply:
Alright. Sorry I'm a little inexperienced with driver development on Zephyr. Wanted to make sure I have the right idea here. Would I just declare a meas,ms5607-xxx and meas,ms5611-xxx yaml files where xxx is the bus protocol? Then in each yaml specify the corresponding chip. In the driver source file the COND_CODE macro I have to modify the check to use a macro like `DT_NODE_HAS_COMPAT` to determine which compatible is being used for determining coefficients. Or did you have something else in mind?